third_party/html5lib.BUILD

```diff
@@ -9,9 +9,44 @@ exports_files(["LICENSE"])
 
 py_library(
     name = "org_html5lib",
-    srcs = glob(["html5lib/**/*.py"]),
-    srcs_version = "PY2AND3",
-    deps = [
-        "@org_pythonhosted_six",
+    srcs = [
```

Comment:
Any particular reason for the explicit file listing replacing the glob?  Was there stuff we wanted to exclude in particular?

Reply:
I don't want to have things like html5lib testonly files schlepped into TensorBoard's pip release archives.

Reply:
Fair enough.  Is there any chance those could be blacklisted (via glob excludes) so we don't need the long whitelist?

Reply:
I like whitelists with BUILD files especially when we're redistributing third party content in this manner.